webapp/app/templates/eligibility/display_success.html

```diff
@@ -14,8 +14,8 @@
                 <img src="{{ url_for('static', filename='icons/success_icon.svg') }}" alt="{{ _('success_icon') }}"/>
             </div>
             <div class="col-10">
-                <h1>{{ render_info.step_title }}</h1>
-                <p class="result-text">{{ error_text }} {{ render_info.step_intro }}</p>
+                <h1 class="headline">{{ render_info.step_title }}</h1>
+                <p class="result-text m-0">{{ error_text }} {{ render_info.step_intro }}</p>
```

Comment:
I am wondering, why do we need the m-0 here and do not set the margin directly in the result-text?